<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>668</width>
    <height>401</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>668</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>21</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="Gate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>371</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Gesamt-Widget der Progressbar */
QProgressBar {
    font-size: 16px;
    border: rgba(0, 0, 0, 0.19);
    background-color: rgba(0, 0, 0, 0.19);  /* optional */
}

/* Der Fortschritts-Chunk */
QProgressBar::chunk {
    background-color:rgb(45, 54, 55);

}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::Direction::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="motorspeed">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Motorspeed: %</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remote_a">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="remote_b">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remote_c">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remote_d">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QToolButton" name="light_barrier">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Barrier</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light_barrier_interrupted">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string>interrupted</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="states">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simulator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
